### ISSUE TITLE
News: Update paint stroke to not be clipped and looks vectorized

### ISSUE THREAD

**Issue by jasmussen** [WordPress/wporg-news-2021#422]
Let me know if this wasn't the right place to add an issue, and feel free to move it if the case.

The brush stroke on the landing page of [News](https://wordpress.org/news/) gets clipped in responsive widths:

![wide](https://github.com/WordPress/wporg-main-2022/assets/1204802/0ec07ae4-9d62-40a9-bf3a-fe240fe8bc42)
![medium](https://github.com/WordPress/wporg-main-2022/assets/1204802/65cc5971-0eeb-4958-9c7d-f0adcc36f11d)
![small](https://github.com/WordPress/wporg-main-2022/assets/1204802/5d018aae-1b81-4541-b795-6a780ea6fac6)

Additionally it looks "crystalline" and vectorized, in contrast to the brush on the homepage which looks like real paint. 

Here's a proposal to update it:

![before-after](https://github.com/WordPress/wporg-main-2022/assets/1204802/f9a985c7-df0b-43ff-b0c6-6f77ef8395bf)

Finding the right brush stroke is non trivial. I wasn't able to find in [the News figma](https://www.figma.com/file/O9I8Jl7kaUMTX39LlRGbGO/News?type=design&node-id=14453%3A20000&mode=design&t=CM0HCZC4YUre1Kei-1) a brush stroke that wasn't vectorized, but finding the original long one would be ideal. In absence of that, here's a PNG version of the one in the above example:

[stroke.png.zip](https://github.com/WordPress/wporg-main-2022/files/14977416/stroke.png.zip)

**Comment by ryelle** [comment]
> Let me know if this wasn't the right place to add an issue, and feel free to move it if the case.

News is a separate site & theme from main, I've moved this to [wporg-news-2021](https://github.com/WordPress/wporg-news-2021/).

**Comment by ndiego** [comment]
A great subtle improvement while remaining consistent with the original design. 🚢

**Comment by jasmussen** [comment]
I'm actually not entirely feeling that particular brush stroke. I think the size, fidelity, color, and position is right, but there's something about the older one that had a bit more character. I wonder, @fcoveram if you have thoughts?

**Comment by fcoveram** [comment]
>  I think the size, fidelity, color, and position is right, but there's something about the older one that had a bit more character.

I agree. The new one looks unintentional without expanding endlessly to the right side. It feels too decorative. Perhaps you can try placing it separately from News heading on the right side, similarly to the new homepage's hero.

**Comment by jasmussen** [comment]
The position we can tinker with, yes, but I actually also think we need a better paint stroke! I'll see if I can find the time, but if you happen upon one, go for it!

**Comment by 4thhubbard** [comment]
Based on the work @richtabor is doing to this page, I'll close this for now.
